desk.go

```diff
@@ -134,6 +170,24 @@ func (l *deskLayout) scaleVars(scale float32) []string {
 	}
 }
 
+func (l *deskLayout) Screens() []Screen {
```

Comment:
Because in the case of an embedded desktop - the deskLayout won't have access to x11wm information

So depending on if it is embedded desktop vs regular - we set which interface to use - which is cleaner in the rest of the code to say deskLayout.screens rather than if (embedded) everywhere.

Reply:
Yes I see that the layout contains a Screens type, but I don’t understand why there is the additional implementation of the interface in the layout code here...

Reply:
Because of embedded desktop ... the screens interface from x11WM will not work with embedded desktop mode, so something else has to implement the interface as well.  In this case that something is deskLayout.